CalcualtorLogic.js

```diff
@@ -18,16 +18,140 @@ class CalculatorLogic {
 };
 
 let calculatorLogic = new CalculatorLogic();
-calcualtorLogic.add();
-calcualtorLogic.subtract();
-calcualtorLogic.multiply();
-calcualtorLogic.divide();
+calculatorLogic.add();
+calculatorLogic.subtract();
+calculatorLogic.multiply();
+calculatorLogic.divide();
 
 
 class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.classList.add("wrapper");
+        const resultDiv = document.createElement('div');
+        resultDiv.classList.add("result");
+        resultDiv.id = "result";
+        wrapperDiv.appendChild(resultDiv);
+        this.hostElem.appendChild(wrapperDiv);
```

Comment:
this code `this.hostElem.appendChild(wrapperDiv);` need to be moved to the bottom of the method.